setup.py

```diff
@@ -86,6 +86,46 @@ def get_hostapd():
         sys.exit(hostapd_message)
 
 
+def get_libdbus():
+    """
+    Try to install the libdbus
+    return: None
+    retype: None
+    """
+
+    # install the libdbus-1
+    if not find_library("dbus-1"):
```

Comment:
Thank you both for reviewing!

@blackHatMonkey it seems that the prefix `lib` can be omitted when use `ctypes`:

Here is the output from `ldconfig -p | grep dbus` on my machine:
```shell
root@user:~# ldconfig -p |grep dbus
	libnih-dbus.so.1 (libc6) => /lib/i386-linux-gnu/libnih-dbus.so.1
	libdbusmenu-qt5.so.2 (libc6) => /usr/lib/i386-linux-gnu/libdbusmenu-qt5.so.2
	libdbusmenu-qt.so.2 (libc6) => /usr/lib/i386-linux-gnu/libdbusmenu-qt.so.2
	libdbusmenu-gtk3.so.4 (libc6) => /usr/lib/i386-linux-gnu/libdbusmenu-gtk3.so.4
	libdbusmenu-gtk.so.4 (libc6) => /usr/lib/i386-linux-gnu/libdbusmenu-gtk.so.4
	libdbusmenu-glib.so.4 (libc6) => /usr/lib/i386-linux-gnu/libdbusmenu-glib.so.4
	libdbus-1.so.3 (libc6) => /lib/i386-linux-gnu/libdbus-1.so.3
	libdbus-1.so (libc6) => /usr/lib/i386-linux-gnu/libdbus-1.so
	libdbus-glib-1.so.2 (libc6) => /usr/lib/i386-linux-gnu/libdbus-glib-1.so.2
	libdbus-glib-1.so (libc6) => /usr/lib/i386-linux-gnu/libdbus-glib-1.so
```

Here is the test code from interactive session:
we need only give the middle part of the library name.

```shell
>>> from ctypes.util import find_library
>>> find_library("dbus-1")
'libdbus-1.so.3'
>>> find_library("dbus-1-dev")
>>> find_library("libdbus-1")
>>> find_library("libdbus-glib-1")
>>> find_library("dbus-glib-1")
'libdbus-glib-1.so.2'
```